java/BUILD.bazel

```diff
@@ -231,12 +231,14 @@ genrule(
     srcs = [
         "//java:ray_dist_deploy.jar",
         "//java:gen_maven_deps",
+        "//:core_worker_library_java",
     ],
     outs = ["ray_java_pkg.out"],
     cmd = """
         WORK_DIR=$$(pwd)
         rm -rf $$WORK_DIR/python/ray/jars && mkdir -p $$WORK_DIR/python/ray/jars
         cp -f $(location //java:ray_dist_deploy.jar) $$WORK_DIR/python/ray/jars/ray_dist.jar
+        cp -f $(location //:core_worker_library_java) $$WORK_DIR/python/ray/jars/
```

Comment:
why need to copy this file? it's also packaged in the jar file. We can copy it to session dir, just like other temporary files.

Reply:
As explained above, All files are prepared once in RunManager. This `cp` is used for python runner to prepare the file.